meeteval/wer/api.py

```diff
@@ -0,0 +1,179 @@
+import glob
+import logging
+import re
+from pathlib import Path
+
+import meeteval.io
+from meeteval.wer.wer import ErrorRate
+
+__all__ = [
+    'cpwer',
+    'orcwer',
+    'mimower',
+    'tcpwer',
+]
+
+
+def _load_texts(
+        reference_paths: 'list[str]',
+        hypothesis_paths: 'list[str]',
+        regex,
+        reference_sort: 'bool | str' = False,
+        hypothesis_sort: 'bool | str' = False,
+        file_format=None,
+) -> 'tuple[meeteval.io.SegLST, meeteval.io.SegLST]':
+    """Load and validate reference and hypothesis texts.
+
+    Validation checks that reference and hypothesis have the same example IDs.
+    """
+
+    # Normalize and glob (for backwards compatibility) the path input
+    def _glob(pathname):
+        match = list(glob.glob(pathname))
+        # Forward pathname if not matched to get the correct error message
+        return match or [pathname]
+
+    def load(paths) -> meeteval.io.SegLST:
```

Comment:
Move this to the top level?

Reply:
Any suggestion for a name in top level? Maybe something like `_maybe_load`?

Reply:
I'm fine with that!

Reply:
Done